src/test/java/org/opensearch/knn/index/LuceneEngineIT.java

```diff
@@ -466,6 +475,200 @@ public void testRadiusSearch_usingScoreThreshold_withFilter_usingCosineMetrics_u
         validateRadiusSearchResults(TEST_QUERY_VECTORS, null, score, SpaceType.COSINESIMIL, expectedResults, COLOR_FIELD_NAME, "red", null);
     }
 
+    @SneakyThrows
+    public void testSQ_whenInvalidBits_thenThrowException() {
+        int bits = -1;
+        expectThrows(
+            ResponseException.class,
+            () -> createKnnIndexMappingWithLuceneEngineAndSQEncoder(
+                DIMENSION,
+                SpaceType.L2,
+                VectorDataType.FLOAT,
+                bits,
+                MINIMUM_CONFIDENCE_INTERVAL
+            )
+        );
+    }
+
+    @SneakyThrows
+    public void testSQ_whenInvalidConfidenceInterval_thenThrowException() {
+        double confidenceInterval = 2.5;
+        expectThrows(
+            ResponseException.class,
+            () -> createKnnIndexMappingWithLuceneEngineAndSQEncoder(
+                DIMENSION,
+                SpaceType.L2,
+                VectorDataType.FLOAT,
+                LUCENE_SQ_DEFAULT_BITS,
+                confidenceInterval
+            )
+        );
+    }
+
+    @SneakyThrows
+    public void testSQ_withByteVectorDataType_thenThrowException() {
```

Comment:
combine all 3 of them in 1 test with name testSQ_withInvalidParam_thenThrowException